moveit_core/utils/src/robot_model_test_utils.cpp

```diff
@@ -57,7 +57,16 @@ moveit::core::RobotModelPtr loadTestingRobotModel(const std::string& robot_name)
 urdf::ModelInterfaceSharedPtr loadModelInterface(const std::string& robot_name)
 {
   boost::filesystem::path res_path(MOVEIT_TEST_RESOURCES_DIR);
-  urdf::ModelInterfaceSharedPtr urdf_model = urdf::parseURDFFile((res_path / robot_name / "urdf/robot.xml").string());
+  std::string urdf_path;
+  if (robot_name == "pr2")
+  {
+    urdf_path = (res_path / (robot_name + "_description") / "urdf/robot.xml").string();
```

Comment:
If you don't intend to add other robots with the PR2-style urdf path, I'd even remove using `robot_name` here and go hardcoded on most of the path. The same below

Reply:
Good point, nothing else seems to have this exact layout. Changed.